crates/rolldown/src/stages/link_stage/mod.rs

```diff
@@ -354,11 +357,17 @@ impl<'a> LinkStage<'a> {
                           stmt_info
                             .referenced_symbols
                             .push(importee_linking_info.wrapper_ref.unwrap().into());
-                          // dbg!(&importee_linking_info.wrapper_ref);
                           stmt_info
                             .referenced_symbols
                             .push(self.runtime.resolve_symbol("__toESM").into());
                           stmt_info.declared_symbols.push(rec.namespace_ref);
+                          rec_namespace_ref_to_stmt_idx.lock().expect("should locked").insert(
```

Comment:
A reverse map(declared symbol -> stmt that declared it) , for now it only stores symbols imported from a cjs module, because for now, esm import stmt still is marked has side effects.